src/UIState/SeeDeviceAddress.cpp

```diff
@@ -4,12 +4,17 @@
 
 #include "SeeDeviceAddress.h"
 
-void SeeDeviceAddress::handleKey(char key) {
-  switch (key) {
-    case 'D':  // Don't save (cancel)
-      returnToMainMenu();
-      break;
-    default:
-      break;
-  };
+#include "../Devices/Ethernet_TC.h"
+#include "../Devices/LiquidCrystal_TC.h"
+#include "Ethernet.h"
+
+void SeeDeviceAddress::loop() {
```

Comment:
Is there a reason to use `loop()` instead of `start()`? It seems that if it doesn't change, then there isn't any need to refresh it many times a second.